### DISCUSSION TITLE
Two questions: One is regarding a type error and the other is about themes.

### DISCUSSION THREAD

**Discussion by githubUser01946** [vladmandic/sdnext#2696] [Q&A]
`TypeError: Script.process_batch() missing 12 required positional arguments: 'enabled', 'mimic_scale',
'threshold_percentile', 'mimic_mode', 'mimic_scale_min', 'cfg_mode', 'cfg_scale_min', 'sched_val',
'separate_feature_channels', 'scaling_startpoint', 'variability_measure', and 'interpolate_phi'`

I have just seen the above in the cmd and wondered if it was anything to worry about?

Also, is there a SD.Next theme that isn't dark that I can use?
I know there are a few themes available but most are ugly and I like the Black/Orange one but wondered if there was a similar one but lighter coloured that people use.

Thanks

**Comment by gohan2091** [comment]
The SD.Next themes are extremely ugly. Not perfect but I am using the Ghostly theme which looks much better IMO. If you want something lighter, try the Dracula Revamped.

**Reply by vladmandic** [reply]
sdnext themes are created by community - which means that YOU can also create one and submit one to be included instead of throwing such wording around.

**Reply by githubUser01946** [reply]
Thanks @gohan2091 I'll have a look at your suggestions. Also, I don't think ugly is the right word to use. Just because you don't like them doesn't make them ugly, just not your taste.

Also, what you call ugly someone else may like. 👍🏻


What about the other issue 🤔  anything to worry about? @vladmandic

**Reply by vladmandic** [reply]
re: other issue - it looks like you're using some extension that has those params as they are not sdnext params.
what seems to be happening is that extension is not sending params to server as initialized, but server is expecting them.

**Comment by gohan2091** [comment]
> Thanks @gohan2091 I'll have a look at your suggestions. Also, I don't think ugly is the right word to use. Just because you don't like them doesn't make them ugly, just not your taste.
> 
> Also, what you call ugly someone else may like. 👍🏻
> 
Well ugly is the word you used (to describe most of the themes). Happy to change it to "unattractive". Yes that's true, people have different tastes. I appreciate a good looking interface and I have some graphic design experience but I am no coder and wouldn't know where to start to make my own theme, besides, I am happy enough with Ghostly and some of the other 3rd party themes. While they aren't perfectly compatible with SD.next, they are 95% there and I have no problems using them. They are much more attractive in my opinion.

**Reply by vladmandic** [reply]
Themes are CSS, if you have graphics design experience that is more than perfect, no coding.

**Comment by gohan2091** [comment]
> Themes are CSS, if you have graphics design experience that is more than perfect, no coding.

My graphic design skills are a hobby and rely on Photoshop with a touch of Illustrator. I have no CSS experience, can't even code a basic website using basic HTML with CSS. I could learn but I am happy enough with some of the other themes to invest any time plus gaming and Stable Diffusion is taking up virtually all my free time. Would be amazing if a theme that looked like Ghosty could be fully compatiable with SD.Next as I think that's a really clean design. Out of my hands, maybe someone reading this can make it happen sometime.

**Reply by vladmandic** [reply]
I'm not even aware of "ghostly", can you at least link to it?

**Comment by gohan2091** [comment]
> I'm not even aware of "ghostly", can you at least link to it?

It's listed in SD.Next, that's how I found it. I haven't searched outside of SD.Next for themes, just tried all the ones in the list. It's: huggingface/remilia/Ghostly

**Reply by vladmandic** [reply]
ok, so its <https://huggingface.co/spaces/remilia/Ghostly>
looks ok, pretty basic. not sure what's the "huge" difference.

**Comment by gohan2091** [comment]
> ok, so its https://huggingface.co/spaces/remilia/Ghostly looks ok, pretty basic. not sure what's the "huge" difference.

That's the one but in dark mode. Indeed it is pretty basic but it's clean and attractive in my opinion. The rounded sliders and buttons, the subtile glow as you hover over elements. The colour scheme works well. I don't recall me ever describing it as a "huge difference" but I do recall calling it "clean" and "much more attractive". Of course everything is individual opinion.

This one is good for a lighter theme: [Dracula Revamped](https://huggingface.co/spaces/freddyaboulton/dracula_revamped)

**Comment by vladmandic** [comment]
i've just added two simple themes similar to ghostly to dev branch, simple-dark and simple-light - grascaled, rounded, min effects

**Reply by gohan2091** [reply]
Appreciate the effort but I don't feel in my honest opinion that there is any comparison between simple-dark and ghosty. While simple-dark is fully compatible with SD.Next which is a good thing because buttons are placed  where they should be, the actual look of the skin "in my opinion" is still unattractive. I can go into specific detail if you want to hear my criticism but anyway I will be using Ghostly for the moment.

**Reply by githubUser01946** [reply]
Thanks :)
